### ISSUE TITLE
NSwag Studio 11.0+ Angular TypeScript file upload body missing

### ISSUE THREAD

**Issue by llusetti** [NSwag/NSwag#802]
I have this WebApi endpoint that was working with NSwag Studio 10.6

`
        [HttpPost("{id}/upload-file")]
        [ProducesResponseType(typeof(List<AllegatoDistintaDto>), 200)]
        [ProducesResponseType(typeof(BlsErrorResult), 400)]
        public IActionResult UploadDocumento(int id, [FromBody]IFormFile postedFile)`

Starting from version 11.0 the generated client does not send the body parameter. I have updated both the packages in the webapi project and the client.

The new generated client is missing these bits, that are back ifI use the 10.6 Studio

`content_.append("postedFile", postedFile.data, postedFile.fileName ? postedFile.fileName : "postedFile");`

and 

`body: content_,`

The question is, am I doing somethign wrong or has the new version changed the way to handle file upload ? 

This is the generated specifications from swagger

```json
"/api/clienti/Distinte/{id}/upload-file": {
      "post": {
        "tags": [
          "Distinte"
        ],
        "operationId": "Distinte_UploadDocumento",
        "consumes": [
          "multipart/form-data"
        ],
        "parameters": [
          {
            "type": "integer",
            "name": "id",
            "in": "path",
            "required": true,
            "x-nullable": false,
            "format": "int32"
          },
          {
            "type": "file",
            "name": "postedFile",
            "in": "formData",
            "x-nullable": true
          }
        ],
        "responses": {
          "200": {
            "description": "",
            "schema": {
              "type": "array",
              "items": {
                "$ref": "#/definitions/AllegatoDistintaDto"
              }
            },
            "x-nullable": true
          },
          "400": {
            "description": "",
            "schema": {
              "$ref": "#/definitions/BlsErrorResult"
            },
            "x-nullable": true
          }
        }
      }
    }

**Comment by RSuter** [comment]
I think this is a regression issue

**Comment by RSuter** [comment]
With the fix I get

    uploadDocumento(id: number, postedFile: FileParameter): Observable<string | null> {
        let url_ = this.baseUrl + "/api/clienti/Distinte/{id}/upload-file";
        if (id === undefined || id === null)
            throw new Error("The parameter 'id' must be defined.");
        url_ = url_.replace("{id}", encodeURIComponent("" + id)); 
        url_ = url_.replace(/[?&]$/, "");

        const content_ = new FormData();
        if (postedFile !== null && postedFile !== undefined)
            content_.append("postedFile", postedFile.data, postedFile.fileName ? postedFile.fileName : "postedFile");
        
        let options_ = {
            body: content_,
            method: "post",
            headers: new Headers({
                "Accept": "application/json"
            })
        };

**Comment by RSuter** [comment]
Should be fixed in v11.1.1, please reopen if the issue has not been solved

**Comment by llusetti** [comment]
It works, thanks.
